anta/cli/exec/utils.py

```diff
@@ -128,6 +128,10 @@ async def collect(device: AntaDevice) -> None:
                     logger.error("Unable to collect tech-support on %s: configuration 'aaa authorization exec default local' is not present", device.name)
                     return
 
+                # TODO: ANTA 2.0.0
+                msg = "[DEPRECATED] Using '--configure' for collecting show-techs is deprecated and will be removed in ANTA 2.0.0"
+                logger.warning(msg)
```

Comment:
I think we should use `warnings.warn()` here so that users can control these warnings with `warnings.filterwarnings()`

Reply:
no because by default warnings.warn deprecation warnings are disabled

Reply:
https://docs.python.org/3/library/warnings.html#default-warning-filter